plugins/services/src/js/events/SDKEndpointActions.js

```diff
@@ -0,0 +1,74 @@
+import { RequestUtil } from "mesosphere-shared-reactjs";
+
+import AppDispatcher from "#SRC/js/events/AppDispatcher";
+import {
+  REQUEST_SDK_ENDPOINTS_SUCCESS,
+  REQUEST_SDK_ENDPOINTS_ERROR,
+  REQUEST_SDK_ENDPOINT_SUCCESS,
+  REQUEST_SDK_ENDPOINT_ERROR
+} from "../constants/ActionTypes";
+
+const SDKEndpointsActions = {
+  fetchEndpoints(serviceId) {
+    const url = `/service/${serviceId}/v1/endpoints`;
+    RequestUtil.json({
+      url,
+      method: "GET",
+      success(xhr) {
+        AppDispatcher.handleServerAction({
+          type: REQUEST_SDK_ENDPOINTS_SUCCESS,
+          data: { serviceId, endpoints: xhr }
```

Comment:
Do we need to pass the xhr Object as part of the dispatch payload? Usually we only need to do that on the error case. The payload I'd expect to see here for endpoints would be more like `endpoints: RequestUtil.parseResponseBody(xhr)`

Reply:
The parseResponseBody was created for responses that have a xhr.responseJSON obj or a xhr.responseText obj. In this case we return as a body the array of the endpoints so the parseResponseBody in the RequestUtil won't work for us and we can return the data immediately. 

- From the RequestUtil.parseResponseBody(xhr)
```
var responseJSON = xhr.responseJSON;
    var responseText = xhr.responseText;

    if (responseJSON) {
      return responseJSON;
    }
```

Reply:
Renamed to "endpoints"